### PR TITLE
added html and css file

### PR DESCRIPTION
**Pull request by Kossi-stack**
I am requesting a review of my Portfolio setup and mobile version skeleton.

### CODE REVIEW

**Review by hemantso — CHANGES_REQUESTED** [review]
Hello @Kossi-stack 👋
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a linter setup. Please follow the guidelines carefully. To setup linter follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

------

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Kossi Fioklou Portfolio</title>
+</head>
+<body>
+    <!--This is the section for the header-->
+      <nav id="nav-bar">
```
Comment (AshakaE) [inline index.html:13]: - [ ] Kindly use a `header` tag to wrap all elements here, in this case you used a `nav` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <div id="logo">
+            <a href="#">
+                <img src="My_Logo.png" alt="logo">
+            </a> 
+        </div>
+          <div class="nav-buttons">
+            <a href="#works" class="nav-button">Portfolio</a>
+            <a href="#about" class="nav-button">About</a>
+            <a href="#contact" class="nav-button">Contact</a>
+          </div>
```
Comment (AshakaE) [inline index.html:23]: - [ ] For the mobile version of the portfolio, it is only required that you create a menu icon, and wrap it in a `nav` tag. The menu icon should be a link as well
_Hint - fontawesome icons_

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Kossi-stack , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Please fix the repeating background image issue
_Hint CSS no-repeat_

_Check the comments under the review._

###  Optional suggestions

- [ ] Please organize your image files into an assets folder and reference them from there.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @Kossi-stack,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by Kossi-stack** [comment]
> 
> # Approved 👏
> 
> Hi @Kossi-stack,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

Good morning @arslanbisharat 

After you have approved my project yesterday, I was not able to submit it because it still showing changes required
Please can you check it at your side and reach out to me.

Thanks 😄

**Pull request merged**